README.md

```diff
@@ -24,7 +24,7 @@ php composer.phar require --dev cakephp/debug_kit "~3.0"
 
 * [Load the plugin](http://book.cakephp.org/3.0/en/plugins.html#loading-a-plugin)
 ```php
-Plugin::load('DebugKit', ['bootstrap' => true]);
+Plugin::load('DebugKit', ['bootstrap' => true, 'routes' => true]);
```

Comment:
After looking into it, it doesn't seem possible as-is because `Plugin::routes()` requires `'routes' => true`, so trying to load them individually later doesn't work, making this PR moot since it would still require running `Plugin::bootstrap('DebugKit')` in order to load the routes.